superset/db_engine_specs/postgres.py

```diff
@@ -89,3 +93,28 @@ def convert_dttm(cls, target_type: str, dttm: datetime) -> Optional[str]:
             dttm_formatted = dttm.isoformat(sep=" ", timespec="microseconds")
             return f"""TO_TIMESTAMP('{dttm_formatted}', 'YYYY-MM-DD HH24:MI:SS.US')"""
         return None
+
+    @staticmethod
+    def get_extra_params(database: "Database") -> Dict[str, Any]:
+        """
+        For Postgres, the path to a SSL certificate is placed in `connect_args`.
+
+        :param database: database instance from which to extract extras
+        :raises CertificateException: If certificate is not valid/unparseable
+        :raises JSONDecodeError: If database extra json payload is unparseable
+        """
+        try:
+            extra = json.loads(database.extra or "{}")
+        except json.JSONDecodeError as ex:
+            logger.error(ex)
+            raise ex
+
+        if database.server_cert:
```

Comment:
Looking in the database and I can't find this property / column (?)

Reply:
Hmm, this is my create script for `dbs`:
```sql
CREATE TABLE IF NOT EXISTS "dbs" (
	created_on DATETIME NOT NULL,
	changed_on DATETIME NOT NULL,
	id INTEGER NOT NULL,
	database_name VARCHAR(250) NOT NULL,
	sqlalchemy_uri VARCHAR(1024) NOT NULL,
	created_by_fk INTEGER,
	changed_by_fk INTEGER,
	password BLOB,
	cache_timeout INTEGER,
	extra TEXT,
	select_as_create_table_as BOOLEAN,
	allow_ctas BOOLEAN,
	expose_in_sqllab BOOLEAN,
	force_ctas_schema VARCHAR(250),
	allow_run_async BOOLEAN,
	allow_dml BOOLEAN,
	verbose_name VARCHAR(250),
	impersonate_user BOOLEAN,
	allow_multi_schema_metadata_fetch BOOLEAN,
	allow_csv_upload BOOLEAN DEFAULT 1 NOT NULL,
	encrypted_extra BLOB,
	server_cert BLOB,
	allow_cvas BOOLEAN,
	uuid NUMERIC(16),
	PRIMARY KEY (id),
	CHECK (select_as_create_table_as IN (0, 1)),
	CHECK (allow_ctas IN (0, 1)),
	CHECK (expose_in_sqllab IN (0, 1)),
	CHECK (allow_run_async IN (0, 1)),
	CHECK (allow_dml IN (0, 1)),
	CHECK (impersonate_user IN (0, 1)),
	CHECK (allow_multi_schema_metadata_fetch IN (0, 1)),
	CHECK (allow_csv_upload IN (0, 1)),
	CHECK (allow_cvas IN (0, 1)),
	CONSTRAINT uq_dbs_uuid UNIQUE (uuid),
	FOREIGN KEY(changed_by_fk) REFERENCES ab_user (id),
	FOREIGN KEY(created_by_fk) REFERENCES ab_user (id),
	UNIQUE (database_name)
);
```